app/models/concerns/imageable.rb

```diff
@@ -17,26 +17,26 @@ def show_image(image_field)
 
   def custom_image(image_field, width, height)
     image = send(image_field.to_sym)
-    image.analyze unless image.analyzed
```

Comment:
I think if this wasn't working we would be seeing an error in production since this code is in production.  I thin issue ci has to do with the s3 call that is required by this code which probably fails in the ci environment but not locally or in the production environment.  I think really if we want to make these tests pass we need to add fixtures that work.  One easy way would be use wrap such test in VCR calls.